views/authenticated.ejs

```diff
@@ -18,9 +18,9 @@ With ESFiddle you can sample ES6 (ECMAScript 2015) and ES7 (ECMAScript 2016) wit
     <div class="header">
       <nav>
         <div class="logo"><a href="/"><img src="/images/logo.png"/></a></div>
+        <span class="loginBtns"><a href="/">Home</a></span>
         <span class="loginBtns only-loggedout"><a href="/auth/github">Login</a></span>
         <span class="loginBtns only-logged"><a href="/github/logout">Logout</a></span>
-        <span class="loginBtns only-logged-profile"><a href="/github/myProfile">Profile</a></span>
```

Comment:
I don't think we should remove actions from the navbar maybe just remove the link and put a hash tag in it instead of the full path, what do you think?